frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -0,0 +1,408 @@
+                aria-labelledby="nav-charts-tab"
+                v-if="selectedTab == 'charts'"
+              >
+                <omegaup-user-charts
+                  v-bind:data="charts"
+                  v-bind:username="profile.username"
+                  v-bind:periodStatisticOptions="this.periodStatisticOptions"
+                  v-on:emit-update-period-statistics="
+                    (profileComponent, categories, data) =>
+                      onUpdatePeriodStatistics(
+                        profileComponent,
+                        categories,
+                        data,
+                      )
+                  "
+                  v-bind:aggregateStatisticOptions="
+                    this.aggregateStatisticOptions
+                  "
+                  v-on:emit-update-aggregate-statistics="
+                    (profileComponent) =>
+                      onAggregateStatistics(profileComponent)
+                  "
+                  v-if="charts"
+                ></omegaup-user-charts>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+a:hover {
+  cursor: pointer;
+}
+th.numericColumn {
+  text-align: right;
+}
+[data-user-profile-root] {
+  font-size: 1rem;
+}
+</style>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import { Chart } from 'highcharts-vue';
+import country_Flag from '../CountryFlag.vue';
+import user_BasicInfo from './BasicInfov2.vue';
+import user_Username from './Username.vue';
+import user_Charts from './Chartsv2.vue';
+import badge_List from '../badge/List.vue';
+import common_GridPaginator from '../common/GridPaginator.vue';
+import { types } from '../../api_types';
+import * as ui from '../../ui';
+import { Problem, ContestResult } from '../../linkable_resource';
+
+@Component({
+  components: {
+    'omegaup-user-basicinfo': user_BasicInfo,
+    'omegaup-user-username': user_Username,
+    'omegaup-user-charts': user_Charts,
+    'omegaup-badge-list': badge_List,
+    'omegaup-grid-paginator': common_GridPaginator,
+    'omegaup-countryflag': country_Flag,
+  },
+})
+export default class UserProfile extends Vue {
+  @Prop() data!: types.UserProfileDetailsPayload;
+  @Prop() profileBadges!: Set<string>;
+  @Prop() visitorBadges!: Set<string>;
+  profile = this.data.profile;
+  contests = this.data.contests
+    ? Object.values(this.data.contests)
+        .map((contest) => {
+          const now = new Date();
+          if (contest.place === null || now <= contest.data.finish_time) {
+            return null;
+          }
+          return new ContestResult(contest);
+        })
+        .filter((contest) => !!contest)
+    : [];
+  charts = this.data.stats;
+  T = T;
+  columns = 3;
+  selectedTab = 'badges';
+  get createdProblems(): types.Problem[] {
```

Comment:
esto está regresando `Problem[]`, no `types.Problem[]`:

```suggestion
  get createdProblems(): Problem[] {
```